src/pages/dashboard/map-container.scss

```diff
@@ -32,41 +33,79 @@ $death-color: #404856;
   }
 }
 
-#map-dek {
+.map-dek {
   width: 100%;
   display: flex;
   flex-direction: column;
   align-items: center;
 
   @media screen and (max-width: 500px) {
-    * {
-      text-align: left;
-    }
+    align-items: initial;
   }
+}
 
-  .label {
-    text-align: center;
+%map-toggle-active {
+  color: black;
+  border-color: black;
+  font-weight: 500;
+}
+%map-toggle-inactive {
+  color: lightgray;
+  border: solid lightgray 2px;
+  font-weight: initial;
+}
+
+.map-style-button {
+  cursor: pointer;
+  display: flex;
+  flex-direction: row-reverse;
+  width: 100%;
+  span {
+    @extend %map-toggle-inactive;
+    display: inline-block;
+    font-size: 14px;
+    margin: -1px;
+    padding: 2px 15px;
+    &:nth-child(2) {
+      border-radius: 8px 0 0 8px;
+      border-right-color: black !important;
+    }
+    &:nth-child(1) {
+      border-radius: 0 8px 8px 0;
+    }
+    &.active {
+      @extend %map-toggle-active;
+    }
```

Comment:
@goleary I moved/renamed these styles to `dashboard.scss` since they're used elsewhere on the page, not just for the map. Might need to update this feature branch with what's already merged to `dashboard-gatsby`.

See https://github.com/COVID19Tracking/website/pull/511/files#diff-d99b5a549ac78593a54c2878d296386c.